README.markdown

```diff
@@ -22,3 +22,51 @@ long the outcome is distributed under the same license. Otherwise, you must
 obtain a [commercial license](./LICENSE-COMMERCIAL) that removes such
 restrictions. Read more about our licensing approach
 [here](https://www.sourcemeta.com/licensing/).
+
+Example
+--------------
+
+```cpp
+#include <sourcemeta/jsontoolkit/json.h>
+#include <sourcemeta/jsontoolkit/jsonpointer.h>
+#include <sstream>
+#include <iostream>
+
+int main()
+{
+    // creating json using parse
+    sourcemeta::jsontoolkit::JSON _template_ =
+        sourcemeta::jsontoolkit::parse(R"JSON({
+            "name" : "xyz" ,
```

Comment:
The convention in JSON is no space before `:` and no space before `,`

Reply:
Also, if you will propose a JSON document that describes a person with name, age, etc you should use more appropriate values like `John Doe` instead of `xyz`, which is meaningless and doesn't match the context of the example